.github/workflows/functional.yml

```diff
@@ -111,6 +111,13 @@ jobs:
           make create-kind-cluster KIND_KUBE_CONFIG=${{ github.workspace }}/kube-${{ github.run_id }} ${{ ! contains(matrix.k8s-version, 'latest') && 'KIND_IMAGE=kindest/node:v${k8s_version}' || '' }}
           echo "KUBECONFIG=${{ github.workspace }}/kube-${{ github.run_id }}" >> "$GITHUB_ENV"
 
+
+      - name: Wait for release to exist
```

Comment:
This is just used in conformance tests to ensure that we have published our release before we run conformance tests on it and create a results file for that release.